gcloud/_helpers.py

```diff
@@ -238,3 +241,35 @@ def __init__(self, project=None, implicit=False):
 
 
 _DEFAULTS = _DefaultsContainer(implicit=True)
+
+
+class _ClientProxy(object):
```

Comment:
If we don't use a proxy, then we have to re-weld the client / connection back into all the "real" domain classes.

I note that the docstring is stale here.  I will fix that.